sub-01/ses-01/eeg/sub-01_ses-01_space-CapTrak_coordsystem.json

```diff
@@ -0,0 +1,24 @@
+{
```

Comment:
Same comments as previously. "sub-01" has nothing do to at the root of the project. If needed for included tests, it needs to be nested appropriately. Else, it needs to be excluded from the repo.

Reply:
yeah, I initially kept it there so that devs could easily pull the repo and for example run the app file straight away. but this project has reached the point where this can be excluded.